packaging_ignore.yml

```diff
@@ -0,0 +1,3 @@
+base:
```

Comment:
Can we move this into `.github/workflows` and update workflow definitions? It appears that only the workflows will actually se this file. I do not like to see configuration files at the root directory of a project if possible.

Reply:
Moved